index.html

```diff
@@ -0,0 +1,233 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+
+</head>
+<body>
+  
+
+    <header class="nav">
+        <a href="" class="m-logo" >My Logo</a>
+    
+         <nav>
+           <ul>
+             <li class="link-right">
+                <a href="/someurl" class="inline"><b><img src="images/men.PNG" alt="ss"></b></a>
+             </li>
+        
+           </ul>
+        </nav>
+    </header>
+
+    <section class="section1">
+          <img src="images/l1.PNG" alt="abc" class="l1">
+          <img src="images/m.PNG" alt="abc" class="m">
+          <img src="images/o.PNG" alt="abc" class="m kap">
+
+          <img src="images/r6.png"  alt="abc" class="r6" >
+          <img src="images/e3.png"  alt="abc" class="image3 dkl" >
+    </section>
+
+
+     <section class="section2">
+          <h2 class="h2 imelex">I’m Aziz <br>Glad to see you!</h2>
+                 <h4  class="sec3p1 sec3p11">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
+
+
+          <h5 class="ets" >ETS CONNECT</h5>
+        <ul class="social-menue">
+          <li><img src="images/twitter.PNG" alt="abc"  class="lsls"></li>
+          <li><img src="images/linkedinicon.PNG" alt="abc" class="lsls"></li>
+          <li> <img src="images/medium.PNG" alt="abc" class="lsls"></li>
+          <li> <img src="images/github.PNG" alt="abc" class="lsls"></li>
+          <li><img src="images/angellist.PNG" alt="abc" class="lsls"></li>
+        </ul>
+      </section>
+     
+
+     <section class="section3">
+          
+          <img src="images/c.png" alt="ada" class="imagesec4">
+         
+            <h2 class="tonic tonic1">Tonic</h2>
+               <ul>
+                 <li class="sec3rlink2">
+                    <a href="/someurl" class="sec3rlink31 ils">CANOPY</a>
+                     <b class="sec3p">.</b>
+                     <a href="/someurl" class="sec3rlink31">Backen Dev</a>
+                      <b class="sec3p">.</b>
+                    <a href="/someurl" class="sec3rlink31">2015</a>
+                  </li>
+  
+               </ul>
+
+
+         <p class="sec3p1"> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+
+            <ul>
+              <li class="sec3rlink23">
+                   <a href="/someurl" class="sec3rlink31 slink">HTML</a>
+                   <a href="/someurl" class="sec3rlink31 slink">CSS</a>
+                   <a href="/someurl" class="sec3rlink31  slink">JAVASCRIPT</a>
+               </li>
+            </ul>
+          <button type="submit" class="bb sec3rbutton sec3rlink4"><span class="d">See Projects</span></button>
+
+   </section> 
+
+
+
+
+
+   
```

Comment:
- [ ] Please remove these white spaces between these sections also. As I said they make the codebase harder to read 👍